src/Build/BackEnd/Components/RequestBuilder/RequestBuilder.cs

```diff
@@ -1278,6 +1278,15 @@ private void ConfigureWarningsAsErrorsAndMessages()
                 }
 
                 ISet<string> warningsAsMessages = ParseWarningCodes(project.GetPropertyValue(MSBuildConstants.WarningsAsMessages));
+                ISet<string> noWarn = ParseWarningCodes(project.GetPropertyValue(MSBuildConstants.NoWarn));
```

Comment:
Why do this in code instead of in common.targets?

Reply:
I can move it there instead. When I was look for where I want to make a change, I sometimes forget to include .targets files in the search.

Reply:
Actually, after change waves come in, will there be an easy way to read in an environment variable and use it to determine whether a property should be reset?

Reply:
Especially keeping in mind that it would have to consider if the user disabled an earlier change wave.

Reply:
Yes, we will need change wave observation in properties. Not via environment variable but explicit change-wave checks.